public/static/locales/en/profile.json

```diff
@@ -2,7 +2,8 @@
   "Profile": {
     "myProfile": {
       "memberSince": "Member since {date}",
-      "userDescription": "{bio}"
+      "userDescription": "{bio}",
```

Comment:
`userDescription` seems unnecessary and can be removed, even though this does not relate to the changes in the PR itself